CHANGELOG.md

```diff
@@ -11,17 +11,30 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
+## [2.3.0]
```

Comment:
Maybe we could add that source factors changed from concat to sum under a `Changed` section?

Reply:
Maybe also worth pointing out that `load_models`'s return types have changed as this is kind of the inference API some may interact with.

Reply:
done!